charts/flyte-core/values-gcp.yaml

```diff
@@ -254,6 +255,15 @@ configmap:
       template: "{{ domain }}"
 
   core:
+    # -- follows the structure specified [here](https://pkg.go.dev/github.com/flyteorg/flytepropeller/manager/config#Config).
+    manager:
+      pod-application: "flytepropeller"
+      pod-template-container-name: "flytepropeller"
+      pod-template-name: "flytepropeller-template"
+      scan-interval: 10s
+      shard:
+        type: Hash
+        pod-count: 3
```

Comment:
Also no need to specify them at all for values-*.yaml... it's enough to specify the defaults in values.yaml

Reply:
Sure, removed.